### PR TITLE
(fix) orders are not removed from tracking after being cancelled

### PR DESCRIPTION
**Pull request by veecos2**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:

**A description of the changes proposed in the pull request**:
Calls the stop tracking order in Binance and coinbase markets after order cancellation so that they are removed from tracked orders. This prevents the Pure MM strategy from crashing when I stop it.

### CODE REVIEW

File: wings/market/binance_market.pyx [hunk 1]
```diff
@@ -1065,6 +1065,7 @@ cdef class BinanceMarket(MarketBase):
         return cancel_result
 
     cdef c_cancel(self, str symbol, str order_id):
+        self.c_stop_tracking_order(order_id)
```
Comment (michaelc9) [inline wings/market/binance_market.pyx]: Orders are removed from tracking in ```execute_cancel```, only if the API request to cancel succeed.

File: wings/market/coinbase_pro_market.pyx [hunk 2]
```diff
@@ -776,6 +776,7 @@ cdef class CoinbaseProMarket(MarketBase):
         return order_id
 
     cdef c_cancel(self, str symbol, str order_id):
+        self.c_stop_tracking_order(order_id)
```
Comment (michaelc9) [inline wings/market/coinbase_pro_market.pyx]: Ditto

File: wings/market/binance_market.pyx (hunk 1 above)
Comment (veecos2) [reply wings/market/binance_market.pyx]: That's not happening in the current version. Its still present in inflight orders after succesful cancellation on binance. Let's discuss.

**Comment by martinkou** [comment]
stop tracking should in general never happen in `c_cancel()` - since that's just the start of the cancel operation. Closing this one.

**Pull request closed without merging**